packages/cf-deploy-config-sub-generator/test/__snapshots__/app.test.ts.snap

```diff
@@ -379,21 +378,14 @@ exports[`Cloud foundry generator tests Validate new managed approuter is added w
 {
   "builder": {
     "customTasks": [
-      {
-        "afterTask": "replaceVersion",
-        "configuration": {
-          "appFolder": "webapp",
-          "destDir": "dist",
-        },
```

Comment:
Seems to be covered by standard tasks 
https://sap.github.io/ui5-tooling/v3/pages/Builder/#standard-tasks